executor/executor_test.go

```diff
@@ -236,6 +240,13 @@ func (s *testSuite) TestAdmin(c *C) {
 	r, err = tk.Exec("admin check table admin_test")
 	c.Assert(err, NotNil)
 
+	os.Setenv("TIDB_CHECK_BEFORE_DROP", "1")
+	r, err = tk.Exec("drop table admin_test")
```

Comment:
How can we know the admin check is executed?

Reply:
You use gofail to inject an error in executeDropTable.

Reply:
`admin check table admin_test` must be error, so `drop table admin_test` must be error to know `admin check` is executed.